### PR TITLE
Added new line to README

### PR DESCRIPTION
**Pull request by kristindickerson**
This is a test to learn github. Please do not merge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # OTIC-slugpen
 Program for splitting multi-penetration heat flow raw data file into individual penetrations for processing.
+
+New version with App Designer on it's way...
+
```
Comment (dmwinslow) [inline README.md:5]: Good idea adding a new line at the end of the file.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,5 @@
 # OTIC-slugpen
 Program for splitting multi-penetration heat flow raw data file into individual penetrations for processing.
+
+New version with App Designer on it's way...
```
Comment (dmwinslow) [inline README.md:4]: Maybe say more?

**Review by dmwinslow — COMMENTED** [review]
This looks good, but yes no need to merge it.

**Comment by kristindickerson** [comment]
This was just a test.

**Pull request closed without merging**